templates/vue/src/components/TapestrySidebar.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <div ref="wrapper" :class="['sidebar-container', { closed: closed }]">
+  <div
+    ref="wrapper"
+    :class="['sidebar-container', { closed: closed }]"
+    @mousedown="pauseDragSelect"
```

Comment:
Hey @seanRong is this needed anymore? I tried removing it and it didn't seem to have any effect.